### ISSUE TITLE
In private window, search not function as expected

### ISSUE THREAD

**Issue by madhums** [brave/brave-browser#8643]
## Description 

In private window, search bar is not enabled and does not function when you type something and hit enter.

## Steps to Reproduce

   1. Open Private window by pressing `cmd`+`shift`+`N`
   2. Enter `testing 123`
   3. Hit `enter`

## Expected result:
Should show a dropdown like below in order to be able to search.

<img width="896" alt="Screenshot 2020-03-14 at 00 04 43" src="https://user-images.githubusercontent.com/123815/76665311-8f74ac80-6587-11ea-9d3d-037bf957855a.png">

## Actual result:

Nothing happens, no search dropdown and nothing happens when I hit Enter.

<img width="862" alt="Screenshot 2020-03-11 at 23 26 43" src="https://user-images.githubusercontent.com/123815/76470523-a9c85200-63f0-11ea-9506-f7e3776b9d8b.png">

## Reproduces how often: 

Easily reproduced, happens all the time.

## Brave version (brave://version info)

```
Version 1.4.96 Chromium: 80.0.3987.132 (Official Build) (64-bit)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Don't know
- Can you reproduce this issue with the dev channel? Don't know
- Can you reproduce this issue with the nightly channel? Don't know

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

## Miscellaneous Information:

Using macOS Catalina 10.15.3

**Comment by madhums** [comment]
Anyone? @brave-dev

**Comment by kjozwiak** [comment]
@madhums seems to be due to the DDG extension when enabled in PB. I couldn't reproduce the issue with a clean profile without any extensions on PB. Once I allowed the DDG to work in PB via `brave://extensions/?id=bkdgflcldnnnapblkhphbgpggdiikppg` using `Allow in private`, I could reproduce the issue.

STR:

* install Brave and install https://chrome.google.com/webstore/detail/duckduckgo-privacy-essent/bkdgflcldnnnapblkhphbgpggdiikppg via `brave://extensions`
* Once installed, enable the extension to work in PB via `brave://extensions/?id=bkdgflcldnnnapblkhphbgpggdiikppg` and enabling `Allow in private`
* run through the STR mentioned via https://github.com/brave/brave-browser/issues/8643#issue-579582924

As a work around, maybe disable the DDG extension for PB for now till this has been addressed/fixed?
